### DISCUSSION TITLE
Exposure Time Finder for Micraft 125

### DISCUSSION THREAD

**Discussion by olicr75** [sn4k3/UVtools#853] [Q&A] [answered]
Dear all,

I colleague of mine is using a Micraft 125 (old professional printer) for her research at a university. She is experimenting with some resins and she is in the need for a way to find a good exposure time for her tests. 

I told her to use your software but we couldn't use it as we need to open a file with the extension .mii or .slc in order to prepare the test of multiple exposures. But it is apparently not part of the supported file formats. 

Would it be possible for her to load an already sliced file into UVTools in order to proceed?

If needed, I can provide you with a test file with the .mii extension.

Thank you in advance, 

Best regards,

Olivier.

**Comment by sn4k3** [comment, marked as answer]
It is not possible to use UVtools with those formats. Even if it can load it, those printers do not support the features required to print multi exposure tests.
You are stuck with single print tests.

**Reply by olicr75** [reply]
Hello, and thank you very much for your prompt feedback. 

Okay, I will inform her.

Great tool you made here! Keep up the good work

best regards,
Olivier
